### ISSUE TITLE
Add required documentation for GCC release

### ISSUE THREAD

**Issue by CohenArthur** [Rust-GCC/gccrs#2050]
- [ ] Add changes to `changes.html` (https://gcc.gnu.org/gcc-13/changes.html)
- [ ] Contribute to gcc.gnu.org/wiki for some explanations on how to use the compiler
- [ ] Mention the state of the compiler in `changes.html`
- [ ] Add documentation about our compiler's internal
- More documentation needed?

**Comment by tschwinge** [comment]
That's what I said in yesterday's "gccrs weekly" meeting that I'll take care of.  :+1:

**Comment by CohenArthur** [comment]
> That's what I said in yesterday's "gccrs weekly" meeting that I'll take care of. +1

Yes! And thank you so much :purple_heart: But I didn't want to just write "Thomas said he would do it", that seemed a little too harsh for me haha
